magento/pwa-studio
# Mini Cart V2 - Product link for each product in the mini cart.

## Description

Added product links to the product listings in the new mini cart.

Should be merged only after #2511 is merged.

## Related Issue
Closes [PWA-739](https://jira.corp.magento.com/browse/PWA-739)

### Verification Stakeholders
@jimbo 
@soumya-ashok 
@schensley 

### Verification Steps
1. Go to the deployed app and add an item to the cart.
2. Go to any other page and click on the item in the cart. It should take you to the product's PDP page.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
